assets/js/services/helpers.js

```diff
@@ -1,5 +1,19 @@
 import moment from "moment";
 import { toast } from "bulma-toast";
+import { IIIF_SIZES } from "./global-vars";
+
+/**
+ * Builds IIIF url for placeholder images
+ * @param {string} id This ID is a filesetId
+ * @param {object} imageSize This is an image size type
+ * @return {string} This returns a URL for the image
+ */
+export function buildImageURL(id, imageSize) {
+  const imageHost = id
```

Comment:
How about we re-name this variable `imageHost` so it's a bit more semantic.  The variable value will be used to fill `<img src="VALUE HERE" ...` so why don't we name the variable something like `srcPath`?

Reply:
Sounds good! I'll do that!